Dockerfile

```diff
@@ -1,50 +1,4 @@
-ARG HYRAX_IMAGE_VERSION=3.1.0
-FROM ghcr.io/samvera/hyrax/hyrax-base:$HYRAX_IMAGE_VERSION as hyku-base
-
-USER root
-
-ARG EXTRA_APK_PACKAGES="openjdk11-jre ffmpeg"
-RUN apk --no-cache upgrade && \
-    apk --no-cache add \
-    cmake \
-    libreoffice \
-    libxml2-dev \
-    mediainfo \
-    nodejs \
-    perl \
-    postgresql-client \
-    screen \
-    yarn \
-    $EXTRA_APK_PACKAGES
-
-ARG VIPS_VERSION=8.11.3
-
-RUN set -x -o pipefail \
-    && wget -O- https://github.com/libvips/libvips/releases/download/v${VIPS_VERSION}/vips-${VIPS_VERSION}.tar.gz | tar xzC /tmp \
-    && apk --no-cache add \
-     libjpeg-turbo libpng tiff librsvg libgsf libimagequant poppler-qt5-dev \
-    && apk add --virtual vips-dependencies build-base \
-     libjpeg-turbo-dev libpng-dev tiff-dev librsvg-dev libgsf-dev libimagequant-dev \
-    && cd /tmp/vips-${VIPS_VERSION} \
-    && ./configure --prefix=/usr \
-                   --disable-static \
-                   --disable-dependency-tracking \
-                   --enable-silent-rules \
-    && make -s install-strip \
-    && cd $OLDPWD \
-    && rm -rf /tmp/vips-${VIPS_VERSION} \
-    && apk del --purge vips-dependencies \
-    && rm -rf /var/cache/apk/*
-
-USER app
-
-RUN mkdir -p /app/fits && \
-    cd /app/fits && \
-    wget https://github.com/harvard-lts/fits/releases/download/1.5.0/fits-1.5.0.zip -O fits.zip && \
-    unzip fits.zip && \
-    rm fits.zip && \
-    chmod a+x /app/fits/fits.sh
-ENV PATH="${PATH}:/app/fits"
+FROM ghcr.io/scientist-softserv/dev-ops/samvera:5eb15236 as hyku-base
```

Comment:
@labradford this should be the new sha once base builds https://github.com/scientist-softserv/dev-ops/actions/runs/3422837907/jobs/5700726565